src/IIIF.cc

```diff
@@ -357,14 +360,21 @@ void IIIF::run( Session* session, const string& src )
         float hd = (float)height;
 
         if ( isPCT ){
-          wd = 100.0;
-          hd = 100.0;
+          region[0] = (1.0/region[0]) * wd;
```

Comment:
I think this part is wrong. If percentage values are provided and region[] should be between 0->1.0, then this will output the wrong value. The region array in this section just needs to be divided by 100.0 as in the original code.

Reply:
You are right!

So like this:

```
region[0] = region[0] / 100.0;
```

I've updated the code to reflect this.
Anyway I think the original region should be overwritten so only
values between values between 0 and 1.0 remain, so that other
code can easily depend on that. In contrast to the original code.

Reply:
I agree about keeping the region array as ratios. If I recall, the reason I had it like it was to avoid possible rounding errors, so we'll need to properly test that it now works as it should